templates/batch-llm/README.md

```diff
@@ -0,0 +1,42 @@
+# Batch LLM Inference using Anyscale Jobs and vLLM
+
+This template shows you how to:
+* Run a batch LLM inference workload using [vLLM](https://docs.vllm.ai/en/latest) and [Ray Data](https://docs.ray.io/en/latest/data/data.html)
+* Deploy the batch LLM workload as an Anyscale Service
+
+**Note**: This template can be run from this Workspace.
+
+## Review the Jupyter Notebook
+
```

Comment:
can add a section to run locally in workspace too? 

Basically instruction to run `python batch_llm.py`

Reply:
oh wait. is that jupyter note book code?

Reply:
Make sense, addressed in https://github.com/anyscale/templates/pull/103 .